apps/admin-x-activitypub/src/views/Notifications/Notifications.tsx

```
@@ -353,7 +353,7 @@ const Notifications: React.FC = () => {
                                                         <>
                                                             <div className='mt-2.5 rounded-md bg-gray-100 px-5 py-[14px] group-hover:bg-gray-200 dark:bg-gray-925/30 group-hover:dark:bg-black/40'>
                                                                 <div
-                                                                    dangerouslySetInnerHTML={{__html: stripHtml(group.post?.content || '')}}
+                                                                    dangerouslySetInnerHTML={{__html: stripHtml(group.post?.content || '', ['a'])}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)